integrations/quick-start-guide.md

```diff
@@ -0,0 +1,47 @@
+In this quick start guide, we will go over key features for integration management.
+
+# Integrations Page
```

Comment:
I think you should summarize the main purpose of this page and include a screenshot of how the page would look on a fresh install.
e.g.
> The integrations page is your home for adding and viewing integrations connected to your Aqueduct account. You can navigate to the integrations page by clicking `Integrations` in the left menu bar. This page is divided into two main sections: the [Add an Integration](#add-an-integration "mention") section and the [Connected Integrations](#connected-integrations "mention") section.

The sections are also quite self-explanatory but I think having a short intro about why they might need the section would be useful. E.g. "You can go to the Connected Integrations section to see what integrations have been connected, as well as manage the integration..."

Reply:
The summary sentence looks quite useful, thanks! I feel 'add an integration' and 'connected integrations' are already quite self-explanatory for this overview paragraph so I think we could save the additional explanation in their subsections